### ISSUE TITLE
High latency for Amplify hosted site

### ISSUE THREAD

**Issue by chriskinzel** [aws-amplify/amplify-console#1761]
### Before opening, please confirm:

- [X] I have checked to see if my question is addressed in the [FAQ](https://github.com/aws-amplify/amplify-console/blob/master/FAQ.md).
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-console/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-console/blob/master/CONTRIBUTING.md).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### Describe the bug

We're hosting a SPA site stored in S3 using Amplify and Cloudfront. Our TTFB is routinely 300ms-500ms even when getting CloudFront hits and we have performance mode enabled. Why is this delay so high and is there anything we can do about it?

<img width="373" alt="Screen Shot 2021-04-19 at 8 22 44 PM" src="https://user-images.githubusercontent.com/25732038/115328785-6ccfa280-a14e-11eb-8a25-5e56267ae377.png">

### Expected behavior

Ideally, TTFB below 300ms

### Reproduction steps

Go to site https://www.essentialclinic.ca, open dev tools, inspect network requests.

**Comment by hisham** [comment]
Just to add another data point, I refreshed the page today and see that TTFB was a whopping 1.43 seconds! The x-cache even says 'Hit from cloudfront'. TTFB should be under 100ms if it's a CDN cache hit no?  

<img width="407" alt="Essential Clinic | Canadian Online Hair Loss Clinic 2021-04-20 08-30-47" src="https://user-images.githubusercontent.com/81853/115413799-b94cc980-a1b2-11eb-94fa-ae1069e46d54.png">

**Comment by xiaoxiaosaohuo** [comment]
yes, this is painful, @chriskinzel do you have found how to solve this problem ?

**Comment by tyfuji** [comment]
Hi there, we are experiencing this same problem with a SSR Next.js app. TTFB is regularly 300ms+ on all resources. We are testing using the provided amplify domain, not a custom domain, so the domain shouldn’t be a factor. Can anyone help with this?

**Comment by antoorofino** [comment]
Hi there, we are experiencing this same problem with a SSR Next.js app too. Can anyone help with this?
